project/Bichen07/README.rst

```diff
@@ -0,0 +1,126 @@
+
+#################
+VoronoiHybrid
+#################
+
+*****************
+Basic Information
+*****************
+* **Project Name**: VoronoiHybrid
+* **Github Repo**: https://github.com/Bichen07/VoronoiHybrid
+The Voronoi Diagram is a well know data structure in computetational geometry.
+used for partitioning a plane into regions based on distance to a set of points. 
+Each point (called a seed or site) has a corresponding region consisting of all the points closer to that seed than to any other.

```

Comment:
The lines are still using too many characters in a line.  Use less than 80 characters.

Please pay attention to the text format.  Most editors can be configured to help you stick with a custom plain text width.